README.md

```diff
@@ -29,10 +29,21 @@ Look through the application code. If you have the old Lambda Times (Applied Jav
 ## Self-Study/Essay Questions
 
 - [ ] What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+PropTypes are used to check what exactly is being passed as a prop. It is important for a couple of reasons. Firstly, it helps you to track how you are passing information between components and to make sure that the correct information is being passed where it needs to go. Another important reason to use PropTypes is to help other developers understand what is going on in your code. It makes it easier for someone to pick up and understand your code more quickly.
+
 - [ ] Describe a life-cycle event in React?
+
+A life-cycle event is one of a number of phases that a component goes through as it is being mounted, updated, and eventually unmonunted from a page. It starts with the mounting phase in which a render method is called and the component is being built. Then it moves to the updating phase where setState can be used to update the data and force a re-render. Finally there is unmounting when the component is removed from the screen.
+
 - [ ] Explain the details of a Higher Order Component?
+
+Higher Order Components are functions that can receive a component as an argument, allowing for code to follow DRY principles. They allow for a component to be passed in, add some extra functionality to that component, and then return it. They can also be used to conditionally render data to the screen, such as showing a login page rather than the actual content of the page.
```

Comment:
Interesting tidbit:

"Note that a HOC doesn’t modify the input component, nor does it use inheritance to copy its behavior. Rather, a HOC composes the original component by wrapping it in a container component. A HOC is a pure function with zero side-effects."

https://reactjs.org/docs/higher-order-components.html